examples/trials/benchmarking/automlbenchmark/nni/config.yaml

```diff
@@ -0,0 +1,33 @@
+# put this file in custom input dir or in ~/.config/automlbenchmark directory
```

Comment:
These files should be provided by automlbenchmark repo?

Reply:
This config file is required by the automlbenchmark library because we define NNI as a customized framework. Some parts are not needed for our purpose, though. I have removed them.

Reply:
done